<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1655</width>
    <height>967</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spectral Image Segmentator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="browse_bar">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>680</width>
         <height>37</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="orig_img_header">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>211</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Avenir</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Original Image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="browse_header">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>281</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select image to analyze</string>
       </property>
      </widget>
      <widget class="QLabel" name="orig_img">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>512</width>
         <height>512</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="analyze_btn">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>700</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	
	color: rgb(42, 42, 42);
	background-color: rgb(198, 198, 198);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: rgb(134, 134, 134);
}</string>
       </property>
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
      <widget class="QLabel" name="anal_img_header">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>130</y>
         <width>451</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Avenir</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Analyzed Image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="anal_img">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>170</y>
         <width>512</width>
         <height>512</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_btn">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	
	color: rgb(42, 42, 42);
	background-color: rgb(198, 198, 198);
	border: 0px solid;
}
QPushButton:hover {
	
	background-color: rgb(134, 134, 134);
}</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1655</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
